webapp/src/app.d.ts

```diff
@@ -0,0 +1,132 @@
+
+type Header = pxt.workspace.Header;
+type ScriptText = pxt.workspace.ScriptText;
+type WorkspaceProvider = pxt.workspace.WorkspaceProvider;
+type InstallHeader = pxt.workspace.InstallHeader;
+
+declare namespace pxt.editor {
+
+export interface IEditor {
```

Comment:
Is this being used? The Editor class in srceditor.tsc doesn't seem to implement anything? Not sure an interface is really necessary here.